src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java

```diff
@@ -317,7 +317,7 @@ private boolean isAnyCartridgeAvailable() {
             UtilsGUI.showDialogInfo(
                     MainActivity.this,
                     getString(R.string.no_wherigo_cartridge_available, FileSystem.ROOT,
-                            MainApplication.APP_NAME));
+                            getString(R.string.app_name)));
```

Comment:
I'm not sure why the APP_NAME string was used here. The no_wherigo_cartridge_available string does not seem to contain a format template usage for %2.

Reply:
You are right. Maybe historic.
Please remove the parameter (and check ist).

Reply:
removed and ready for merge from my perspective